pkg/network/tracer/connection/tracer.go

```diff
@@ -581,6 +587,10 @@ func populateConnStats(stats *network.ConnectionStats, t *netebpf.ConnTuple, s *
 	default:
 		stats.Direction = network.OUTGOING
 	}
+
+	if ch != nil {
+		ch.Hash(stats)
+	}
```

Comment:
why don't we check that we didn't hash the connection already?
is such a scenario realistic?

Reply:
In this case we're always converting from the ebpf side to userspace, so the hash will not be done.